db/migrate/20231028090955_create_likes.rb

```diff
@@ -0,0 +1,9 @@
+class CreateLikes < ActiveRecord::Migration[7.0]
+  def change
+    create_table :likes do |t|
+      t.references :author, null: false, foreign_key: { to_table: 'users' }
```

Comment:
- [ ] In keeping with the guidelines given in the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png), the foreign key should be `user_id` and not `author_id` as you have specified in your migration file above ☝️ 

#### Only `Post` should have `author_id` 👍

Reply:
well resolved! ✔️